docs/reST/ref/cursors.rst

```diff
@@ -130,4 +130,100 @@ The following strings can be converted into cursor bitmaps with
+      pygame.SYSTEM_CURSOR_IBEAM       i-beam
+      pygame.SYSTEM_CURSOR_WAIT        wait
+      pygame.SYSTEM_CURSOR_CROSSHAIR   crosshair
+      pygame.SYSTEM_CURSOR_WAITARROW   small wait cursor 
+                                       (or wait if not available)
+      pygame.SYSTEM_CURSOR_SIZENWSE    double arrow pointing 
+                                       northwest and southeast
+      pygame.SYSTEM_CURSOR_SIZENESW    double arrow pointing
+                                       northeast and southwest
+      pygame.SYSTEM_CURSOR_SIZEWE      double arrow pointing
+                                       west and east
+      pygame.SYSTEM_CURSOR_SIZENS      double arrow pointing 
+                                       north and south
+      pygame.SYSTEM_CURSOR_SIZEALL     four pointed arrow pointing
+                                       north, south, east, and west
+      pygame.SYSTEM_CURSOR_NO          slashed circle or crossbones
+      pygame.SYSTEM_CURSOR_HAND        hand
+
+   **Creating a color cursor**
+
+   To create a color cursor, create a ``Cursor`` from a ``hotspot`` and a ``surface``.
+   ``hotspot`` is an (x,y) coordinate that determines where in the cursor the exact point is.
+   The hotspot position must be within the bounds of the ``surface``.
+
+   **Creating a bitmap cursor**
+
+   When the mouse cursor is visible, it will be displayed as a black and white
+   bitmap using the given bitmask arrays. The ``size`` is a sequence containing 
+   the cursor width and height. ``hotspot`` is a sequence containing the cursor 
+   hotspot position. 
+   
+   A cursor has a width and height, but a mouse position is represented by a 
+   set of point coordinates. So the value passed into the cursor ``hotspot`` 
+   variable helps pygame to actually determine at what exact point the cursor 
+   is at.
+   
+   ``xormasks`` is a sequence of bytes containing the cursor xor data masks. 
+   Lastly ``andmasks``, a sequence of bytes containing the cursor bitmask data.
+   To create these variables, we can make use of the 
+   :func:`pygame.cursors.compile()` function.
+
+   Width and height must be a multiple of 8, and the mask arrays must be the 
+   correct size for the given width and height. Otherwise an exception is raised.
+
+   .. attribute:: type
+   
+      | :sl:`Gets the cursor type`
+      | :sg:`type -> string`
+
+      The type will be ``"system"``, ``"bitmap"``, or ``"color"``.
+
+   .. ## pygame.cursors.Cursor.type ##
+
+   .. attribute:: data
+
+      | :sl:`Gets the cursor data`
+      | :sg:`data -> tuple`
+
+      Returns the data that was used to create this cursor object, wrapped up in a tuple.
+
+   .. ## pygame.cursors.Cursor.data ##
+
+   .. versionadded:: 2.0.1
+
+   .. ## pygame.cursors.Cursor ##
+   
 .. ## pygame.cursors ##
+
+Example code for creating and settings cursors. (Click the mouse to switch cursor)
+
+.. literalinclude:: code_examples/cursors_module_example.py
```

Comment:
I wonder if the example can go inside examples/ instead and still be included?

Reply:
I'll give it a shot. Right now there is a cursors example in the examples folder, so the two would need to be merged together.

Reply:
We could have two examples if it makes it a bit clearer. Or the old one can be replaced if it has no value anymore.

Reply:
The old one still has value, since it shows the compiling step from an actual collection of strings. 

I would like to get this merged first, then do that in a subsequent PR.

Reply:
Let's leave it, and perhaps address it later.